_posts/2020-01-26-newpipe-0.18.2-released.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "NewPipe 0.18.2 released: Hotfix for the YouTube-Extractor"
+short: "NewPipe 0.18.1 released"
+date: 2020-01-26 00:00:00 +0100
+categories: [release, pinned]
+author: Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+8 days after the last release and the next one is already out: A hotifx for the Extractor! A change on YouTubes side made it necessary.
+
+## Hotfix
+
+A hotfix release regarding the Extractor usually means that all the older versions of NewsPipe are likely to fail to play YouTube videos - this release is no difference. If you see any Guru Mediation error in the app, please update to this version before you create an issue (and/or verify that you actually have `0.18.2+` installed via hitting the `About` section in the left menu).
```

Comment:
The excerpt shouldn't include any heading. Also it's not `NewsPipe`, it's `NewPipe`.

Reply:
I liked it as a separator from the headline, and also to state what the paragraph is about.

Its not NewsPipe though, correct.